src/create-type.test.js

```diff
@@ -17,11 +17,24 @@ describe('KISS type', () => {
     })
     it('fails on attempted access to unknown props', () => {
       const failsNicely = __.throws(
-        __.typedError(TypeError, __.containsString('unknown'))
+        __.typedError(TypeError, __.containsString('Unknown'))
       )
       __.assertThat(() => type.get.unknown, failsNicely)
       __.assertThat(() => type.set.unknown, failsNicely)
     })
+    it('all props can be retrieved with ramda', () => {
+      const doesNotThrow = __.not(
+        __.throws(
+          __.typedError(TypeError, __.containsString('Unknown'))
+        )
+      )
+      __.assertThat(() =>
+        r.pipe(
+          r.prop('props'),
+          r.values,
+        )(type)
+        , doesNotThrow)
```

Comment:
Let’s extract the pipe into a function so that we get a nice assertion. Maybe sth like `__.assertThat(() => getAllProps(type), doesNotThrow)`?